### ISSUE TITLE
Ambiguity in recurring task due dates if the task is due next year

### ISSUE THREAD

**Issue by pmason314** [jamiebrynes7/obsidian-todoist-plugin#293]
**Describe the bug**

Huge fan of the plugin, thanks for all the effort you put in!  My request is about an ambiguity that occurs if a Todoist task's due date isn't in the current year.  As an example, if I have a query block to get all long term tasks and an annually recurring task that's due March 31 and it's March 15 now, I can't tell if I've checked off the task for this year or accidentally checked it for next year.  Apologies if that's confusing, but without the context clues of other tasks, I can't tell what year the annually recurring task is meant for.  I've included my query block that results in the ambiguity below.

**To Reproduce**

1. I used this query block:
```todoist
name: Long Term TODOs
filter: "!(7 days) & !(no date) & !overdue"
sorting:
  - date
show: 
  - date
  - description
```
2. Create an annually recurring task in Todoist with a due date of March 31st. 
3. In Obsidian, see that the task shows up with the correct due date.
4. Check off/complete this year's instance of the task, then see that next year's instance of the task is due March 31st, without any indication of what year it's for.

**Expected behavior**
I think the easiest way to clear up the ambiguity is to add the year to the date field (where it gets rendered), if the task's due date year isn't the current year.  For the example above, the task should show a due date of "March 31st", then once I check it off the next instance should show up as "March 31st 2025".

This could be a configuration within the plugin's query block (i.e. configure the format of the date you'd like to see rendered).  I think the show-year-if-it's-not-this-year approach is a more useful default behavior, but I'd be happy with either option.

**Screenshots**
I don't think a screenshot is particularly useful here, but happy to provide one if needed.

**Desktop (please complete the following information):**
 - Plugin Version - 1.12.0
 - Obsidian Version - v.1.5.8

**Additional context**
Thank you!

**Comment by jamiebrynes7** [comment]
This should be fixed in the latest (v2.0.0) release!

**Comment by pmason314** [comment]
Just updated and confirmed it's fixed, thank you!
